include/translated-object.php

```diff
@@ -499,6 +533,25 @@ protected function validate_translations( $translations, $id = 0, $context = 'sa
 		/** @phpstan-var array<non-empty-string, positive-int> $translations */
 		return array_merge( array( $lang->slug => $id ), $translations );
 	}
+
+	/**
+	 * Validates and sanitizes multiple translations arrays.
```

Comment:
It doesn't only validate but also flattens the array, thus the name and description could be updated.

Reply:
An idea for the name ?

Reply:
Done in d4df22e6f228c867275cee1871b7fc228421b4ea